client/src/document/index.scss

```
@@ -148,7 +148,6 @@
       a {
         &:link,
         &:visited {
-          color: inherit;
           text-decoration: none;
```

```suggestion
          text-decoration: underline;
        }

        &:hover,
        &:focus {
          text-decoration: none;
        }

        &[href^="#"] {
          color: inherit;
          position: relative;

          &:link,
          &:visited {
            text-decoration: none;
          }

          &:hover,
          &:focus {
            text-decoration: underline;
          }

          &::before {
            color: var(--text-inactive);
            content: "#";
            display: inline-block;
            font-size: 0.7em;
            line-height: 1;
            margin-left: -0.8em;
            text-decoration: none;
            visibility: hidden;
            width: 0.8em;
          }

          &:hover::before {
            visibility: visible;
```
1. For underline, hyperlinks and page anchors have opposite behavior. Hyperlinks have underlines and on hover it is removed. For page anchors underlines are not visible but are shown on hover.
2. The `#` on page anchors is not interactible with `position: absolute`. It works well with same pseudo-element implementation of H1-6 headers.